lib/ansible/inventory/manager.py

```diff
@@ -46,9 +47,14 @@
 
 IGNORED_ALWAYS = [br"^\.", b"^host_vars$", b"^group_vars$", b"^vars_plugins$"]
 IGNORED_PATTERNS = [to_bytes(x) for x in C.INVENTORY_IGNORE_PATTERNS]
-IGNORED_EXTS = [b'%s$' % to_bytes(re.escape(x)) for x in C.INVENTORY_IGNORE_EXTS]
+IGNORED_EXTS = [b'%s$' % to_bytes(re.escape(x))
+                for x in C.INVENTORY_IGNORE_EXTS]
 
-IGNORED = re.compile(b'|'.join(IGNORED_ALWAYS + IGNORED_PATTERNS + IGNORED_EXTS))
+IGNORED = re.compile(
+    b'|'.join(
+        IGNORED_ALWAYS +
+        IGNORED_PATTERNS +
+        IGNORED_EXTS))
```

Comment:
there are also a bunch of style changes in this PR, i would remove those to limit the scope of what needs to be reviewed.

Reply:
oh man i must have formatted it again, just reverted the style changes with the newest commit!